--- a/common/cmake/ALPSCoreConfig.cmake.in
+++ b/common/cmake/ALPSCoreConfig.cmake.in
@@ -145,12 +145,12 @@
  Current C++ compiler ID: ${CMAKE_CXX_COMPILER_ID}
  Current C++ compiler version: ${CMAKE_CXX_COMPILER_VERSION}
  Current C++ compiler: ${CMAKE_CXX_COMPILER}")
-  if (ALPS_FORCE_NO_COMPILER_CHECK)
-    message("WARNING: Compiler mismatch is ignored, as requested")
+ #  if (ALPS_FORCE_NO_COMPILER_CHECK)
+ #   message("WARNING: Compiler mismatch is ignored, as requested")
     set(ALPSCore_COMPILER_MATCH true)
-  else()
-    set(ALPSCore_COMPILER_MATCH false)
-  endif()
+ # else()
+ #   set(ALPSCore_COMPILER_MATCH false)
+ # endif()
 endif()
 if (ALPSCore_COMPILER_MATCH AND ALPSCore_CMAKE_MATCH)
   alpscore_config_main_()
