--- a/src/system.make
+++ b/src/system.make
@@ -1,27 +1,28 @@
 #
 
 # Set compilers
-FC=mpif90
-F77=mpif77
+FC = mpif90
+F77 = mpif77
 
 # Linking flags
-LINKFLAGS= -L/usr/local/lib
-ARFLAGS=
+LINKFLAGS = -L/usr/local/lib
+ARFLAGS =
 
 # Compilation flags
 # NB for gcc10 you need to add -fallow-argument-mismatch
-COMPFLAGS= -O3 $(XC_COMPFLAGS)
-COMPFLAGS_F77= $(COMPFLAGS)
+COMPFLAGS = -O3 $(XC_COMPFLAGS)
+COMPFLAGS_F77 = $(COMPFLAGS)
 
 # Set BLAS and LAPACK libraries
 # MacOS X
-# BLAS= -lvecLibFort
+# BLAS = -lvecLibFort
 # Intel MKL use the Intel tool
 # Generic
-# BLAS= -llapack -lblas
+BLAS = -llapack -lblas
+SCALAPACK_LIB = -lscalapack
 
 # Full library call; remove scalapack if using dummy diag module
-LIBS= $(FFT_LIB) $(XC_LIB) -lscalapack $(BLAS)
+LIBS = $(FFT_LIB) $(XC_LIB) $(SCALAPACK_LIB) $(BLAS)
 
 # LibXC compatibility (LibXC below) or Conquest XC library
 
@@ -38,8 +39,8 @@
 XC_COMPFLAGS = -I/usr/local/include
 
 # Set FFT library
-FFT_LIB=-lfftw3
-FFT_OBJ=fft_fftw3.o
+FFT_LIB = -lfftw3
+FFT_OBJ = fft_fftw3.o
 
 # Matrix multiplication kernel type
 MULT_KERN = default
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,3 @@
+default clean:
+	make -C src $@
+	make -C tools/BasisGeneration $@
--- a/tools/BasisGeneration/system.make
+++ b/tools/BasisGeneration/system.make
@@ -1,27 +1,28 @@
 #
 
 # Set compilers
-FC=mpif90
-F77=mpif77
+FC = mpif90
+F77 = mpif77
 
 # Linking flags
-LINKFLAGS= -L/usr/local/lib
-ARFLAGS=
+LINKFLAGS = -L/usr/local/lib
+ARFLAGS =
 
 # Compilation flags
 # NB for gcc10 you need to add -fallow-argument-mismatch
-COMPFLAGS= -O3 $(XC_COMPFLAGS)
-COMPFLAGS_F77= $(COMPFLAGS)
+COMPFLAGS = -O3 $(XC_COMPFLAGS)
+COMPFLAGS_F77 = $(COMPFLAGS)
 
 # Set BLAS and LAPACK libraries
 # MacOS X
-# BLAS= -lvecLibFort
+# BLAS = -lvecLibFort
 # Intel MKL use the Intel tool
 # Generic
-# BLAS= -llapack -lblas
+BLAS = -llapack -lblas
+SCALAPACK_LIB = -lscalapack
 
 # Full library call; remove scalapack if using dummy diag module
-LIBS= $(FFT_LIB) $(XC_LIB) -lscalapack $(BLAS)
+LIBS = $(FFT_LIB) $(XC_LIB) $(SCALAPACK_LIB) $(BLAS)
 
 # LibXC compatibility (LibXC below) or Conquest XC library
 
@@ -38,8 +39,8 @@
 XC_COMPFLAGS = -I/usr/local/include
 
 # Set FFT library
-FFT_LIB=-lfftw3
-FFT_OBJ=fft_fftw3.o
+FFT_LIB = -lfftw3
+FFT_OBJ = fft_fftw3.o
 
 # Matrix multiplication kernel type
 MULT_KERN = default
--- a/src/Makefile
+++ b/src/Makefile
@@ -97,6 +97,8 @@
 
 clean:
 	rm -f *.o *.mod *~  *.d work.pc* deps.obj.inc
+	rm -f datestamp.f90 makedeps.txt
+	rm -f $(TARGET) ../bin/$(TARGET)
 #	(cd FFT; make -k clean)
 #	(cd utilities; make -k clean)
 
--- a/tools/BasisGeneration/Makefile
+++ b/tools/BasisGeneration/Makefile
@@ -30,3 +30,5 @@
 
 clean:
 	rm -f *.o *.mod *~
+	rm -f datestamp.f90 deps.obj
+	rm -f $(TARGET) ../../bin/$(TARGET)
