--- a/meson.build
+++ b/meson.build
@@ -5,7 +5,7 @@
 cc = meson.get_compiler('c')
 blas_deps = []
 
-if get_option('link_mkl')
+if false
     blas_deps = [cc.find_library('mkl_rt', required : false)]
     if not blas_deps[0].found()
         blas_deps = [dependency('mkl-sdl', required : false)]
@@ -14,9 +14,9 @@
     if host_machine.system() == 'darwin'
         blas_deps = [dependency('Accelerate')]
     else
-        blas_deps = [cc.find_library('openblas', static: get_option('link_blas_statically'), required : false)]
+        blas_deps = [cc.find_library('openblas', static: false, required : false)]
         if not blas_deps[0].found()
-            blas_deps = [dependency(['openblas', 'OpenBLAS'], static: get_option('link_blas_statically'), required : false)]
+            blas_deps = [dependency(['openblas', 'OpenBLAS'], static: false, required : false)]
         endif
     endif
 
@@ -155,7 +155,7 @@
     dependencies: blas_deps,
 )
 
-if get_option('link_mkl')
+if false
     py.extension_module(
         '_scs_mkl',
 
