--- a/pyproject.toml
+++ b/pyproject.toml
@@ -34,7 +34,7 @@
     "Programming Language :: Python :: Implementation :: PyPy",
     "Topic :: Software Development :: Libraries",
 ]
-dynamic = ["version", "description"]
+dynamic = ["version"]
 
 [project.urls]
 Homepage = "https://github.com/more-itertools/more-itertools"
--- a/setup.cfg
+++ b/setup.cfg
@@ -24,3 +24,9 @@
 
 [mypy-tests.*]
 disallow_untyped_defs = false
+
+[options]
+packages = find:
+
+[options.packages.find]
+exclude = debian
