diff -urN gamess.orig/comp gamess/comp
--- gamess.orig/comp	2024-07-23 15:20:28.000000000 +0900
+++ gamess/comp	2025-06-06 15:00:25.125481422 +0900
@@ -2223,7 +2223,9 @@
          if ($GMS_HPC_SYSTEM_TARGET == grace-hopper) then
             set EXTRAOPT="-mcmodel=large $EXTRAOPT"
          else
-            set EXTRAOPT="-mcmodel=medium $EXTRAOPT"
+            if (`uname -m` == x86_64) then
+                set EXTRAOPT="-mcmodel=medium $EXTRAOPT"
+            endif
          endif
          #
          if ($GMS_OPENMP_OFFLOAD == true) set OPT="${OPT} ${OFFLOAD_FLAGS}"
@@ -2288,7 +2290,7 @@
          #
          # ARM64
          #
-         if (`uname -p` == aarch64) set EXTRAOPT="-mcpu=native $EXTRAOPT"
+         if (`uname -m` == aarch64) set EXTRAOPT="-mcpu=native $EXTRAOPT"
          #
          if ($GMS_OPENMP_OFFLOAD == true) set OPT="${OPT} ${OFFLOAD_FLAGS}"
          #
diff -urN gamess.orig/ddi/compddi gamess/ddi/compddi
--- gamess.orig/ddi/compddi	2024-07-23 15:20:28.000000000 +0900
+++ gamess/ddi/compddi	2025-06-06 15:30:05.091020273 +0900
@@ -127,7 +127,7 @@
 #
 # ARM64
 #
-if (`uname -p` == aarch64) then
+if (`uname -m` == aarch64) then
    set MAXCPUS=256
    set MAXNODES=10
 endif
@@ -867,7 +867,7 @@
    #
    set ARCH='-m64'
    if (`uname -p` == ia64) set ARCH=''
-   if (`uname -p` == aarch64) set ARCH='-mcpu=native'
+   if (`uname -m` == aarch64) set ARCH='-mcpu=native'
 
    set CC = 'gcc'
    set CFLAGS = "-DLINUX $ARCH -O3 -fstrict-aliasing -I./"
diff -urN gamess.orig/rungms gamess/rungms
--- gamess.orig/rungms	2024-07-23 15:20:28.000000000 +0900
+++ gamess/rungms	2025-06-06 15:00:56.269045437 +0900
@@ -161,7 +161,7 @@
 set NCPUS=$3    # number of compute processes to be run
 # $4 is treated below.
 set LOGN=$5     # number of cores per logical node
-if($# > 5) then
+if($#argv > 5) then
   if($6 > 0) setenv OMP_NUM_THREADS $6 # number of threads
 endif
 set FSAVE="$7"  # extra files to save, example: F10, F06 etc.
