diff -urN gamess.orig/comp gamess/comp
--- gamess.orig/comp	2023-10-03 06:37:23.000000000 +0000
+++ gamess/comp	2024-01-21 04:39:02.466477002 +0000
@@ -2294,7 +2294,7 @@
          # ARM64 ThunderX2
          if (`uname -p` == aarch64) set EXTRAOPT="-mcpu=native $EXTRAOPT"
          #
-         set EXTRAOPT="-mcmodel=medium $EXTRAOPT"
+         if (`uname -m` == x86_64) set EXTRAOPT="-mcmodel=medium $EXTRAOPT"
          #
          (set echo; gfortran -c -fdefault-integer-8 -g $GMS_FPE_FLAGS $OPT $EXTRAOPT $MODULE_F)
          breaksw
diff -urN gamess.orig/ddi/compddi gamess/ddi/compddi
--- gamess.orig/ddi/compddi	2023-10-03 06:37:23.000000000 +0000
+++ gamess/ddi/compddi	2024-01-21 04:39:02.466477002 +0000
@@ -866,7 +866,7 @@
    #
    set ARCH='-m64'
    if (`uname -p` == ia64) set ARCH=''
-   if (`uname -p` == aarch64) set ARCH='-mcpu=native'
+   if (`uname -m` == aarch64) set ARCH='-mcpu=native'
 
    set CC = 'gcc'
    set CFLAGS = "-DLINUX $ARCH -O3 -fstrict-aliasing -I./"
diff -urN gamess.orig/rungms gamess/rungms
--- gamess.orig/rungms	2023-07-01 06:04:29.000000000 +0000
+++ gamess/rungms	2024-01-21 04:42:23.574370012 +0000
@@ -96,9 +96,9 @@
 #       See also a very old LoadLeveler "ll-gms" for some IBM systems.
 #
 set TARGET=sockets
-set SCR=~/gamess/restart
-set USERSCR=~/gamess/restart
-set GMSPATH=~/gamess
+set SCR=$HOME/scr
+set USERSCR=$HOME/scr
+set GMSPATH=$HOME/share/gamess
 set DEL_OLD=yes
 #
 # Get any MDI-related options and remove them from the argument list
