--- a/c++/triqs/mc_tools/random_generator.cpp
+++ b/c++/triqs/mc_tools/random_generator.cpp
@@ -38,7 +38,7 @@
 namespace triqs {
   namespace mc_tools {
 
-    random_generator::random_generator(std::string const &RandomGeneratorName, uint32_t seed_) {
+    random_generator::random_generator(std::string const &RandomGeneratorName, std::uint32_t seed_) {
       _name = RandomGeneratorName;
 
       if (RandomGeneratorName == "") {
--- a/c++/triqs/mc_tools/random_generator.hpp
+++ b/c++/triqs/mc_tools/random_generator.hpp
@@ -21,6 +21,7 @@
 #include <triqs/utility/first_include.hpp>
 #include "../utility/exceptions.hpp"
 #include "../utility/buffered_function.hpp"
+#include <cstdint>
 #include <cmath>
 #include <string>
 #include <assert.h>
@@ -48,7 +49,7 @@
    *  @param RandomGeneratorName : Name of a boost generator e.g. mt19937, or "" (another Mersenne Twister).
    *  @param seed : The seed of the random generator
    */
-      random_generator(std::string const &RandomGeneratorName, uint32_t seed_);
+      random_generator(std::string const &RandomGeneratorName, std::uint32_t seed_);
 
       random_generator() : random_generator("mt19937", 198) {}
 
