--- a/src/alps/parapack/parapack.C
+++ b/src/alps/parapack/parapack.C
@@ -49,6 +49,9 @@
 #include <boost/filesystem/operations.hpp>
 #include <boost/foreach.hpp>
 #include <boost/regex.hpp>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 
 #include <iostream>
--- a/src/alps/ngs/lib/parapack.cpp
+++ b/src/alps/ngs/lib/parapack.cpp
@@ -53,6 +53,9 @@
 #include <boost/filesystem/operations.hpp>
 #include <boost/foreach.hpp>
 #include <boost/regex.hpp>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 
 #include <iostream>
--- a/test/fixed_capacity/timing_queue.C
+++ b/test/fixed_capacity/timing_queue.C
@@ -28,6 +28,9 @@
 /* $Id: timing_queue.C 691 2004-03-16 15:12:31Z wistaria $ */
 
 #include <alps/fixed_capacity_deque.h>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 #include <deque>
 #include <iostream>
--- a/test/fixed_capacity/timing_stack.C
+++ b/test/fixed_capacity/timing_stack.C
@@ -28,6 +28,9 @@
 /* $Id: timing_stack.C 691 2004-03-16 15:12:31Z wistaria $ */
 
 #include <alps/fixed_capacity_vector.h>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 #include <deque>
 #include <iostream>
--- a/test/fixed_capacity/timing_vector.C
+++ b/test/fixed_capacity/timing_vector.C
@@ -29,6 +29,9 @@
 
 #include <alps/fixed_capacity_vector.h>
 #include <iostream>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 #include <vector>
 
--- a/test/parapack/clone_mpi.C
+++ b/test/parapack/clone_mpi.C
@@ -34,6 +34,9 @@
 #include <iostream>
 #include <boost/filesystem/path.hpp>
 #include <boost/filesystem/operations.hpp>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 
 namespace mpi = boost::mpi;
--- a/test/parapack/filelock_mpi.C
+++ b/test/parapack/filelock_mpi.C
@@ -30,6 +30,9 @@
 #include <iostream>
 #include <boost/filesystem/path.hpp>
 #include <boost/filesystem/operations.hpp>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 
 namespace mpi = boost::mpi;
--- a/test/parapack/worker_mpi.C
+++ b/test/parapack/worker_mpi.C
@@ -32,6 +32,9 @@
 #include <iostream>
 #include <boost/filesystem/path.hpp>
 #include <boost/filesystem/operations.hpp>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 
 namespace mpi = boost::mpi;
--- a/tutorials/alpsize-05-boost/wolff.C
+++ b/tutorials/alpsize-05-boost/wolff.C
@@ -30,6 +30,9 @@
 
 #include <boost/array.hpp>
 #include <boost/random.hpp>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 #include <cmath>
 #include <iostream>
--- a/tutorials/alpsize-06-parameters/wolff.C
+++ b/tutorials/alpsize-06-parameters/wolff.C
@@ -31,6 +31,9 @@
 #include <alps/parameter.h>
 #include <boost/array.hpp>
 #include <boost/random.hpp>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 #include <cmath>
 #include <iostream>
--- a/tutorials/alpsize-07-alea/wolff.C
+++ b/tutorials/alpsize-07-alea/wolff.C
@@ -32,6 +32,9 @@
 #include <alps/parameter.h>
 #include <boost/array.hpp>
 #include <boost/random.hpp>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 #include <cmath>
 #include <iostream>
--- a/tutorials/alpsize-08-lattice/wolff.C
+++ b/tutorials/alpsize-08-lattice/wolff.C
@@ -33,6 +33,9 @@
 #include <alps/lattice.h>
 #include <boost/foreach.hpp>
 #include <boost/random.hpp>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 #include <cmath>
 #include <iostream>
--- a/example/ietl/jacobidavidson2.cpp
+++ b/example/ietl/jacobidavidson2.cpp
@@ -33,6 +33,9 @@
 #include <ietl/interface/ublas.h>
 #include <boost/random.hpp>
 #include <boost/limits.hpp>
+#if !defined(BOOST_TIMER_ENABLE_DEPRECATED)
+#  define BOOST_TIMER_ENABLE_DEPRECATED
+#endif
 #include <boost/timer.hpp>
 
 #include <cassert>
