--- /dev/null
+++ b/Makefile
@@ -0,0 +1,6 @@
+default:
+	make default -C doc
+	make default -C src
+%:
+	make $@ -C doc
+	make $@ -C src
--- a/src/Makefile-dist
+++ b/src/Makefile-dist
@@ -107,18 +107,23 @@
 	${FC} -O3 -openmp -msse3 -fno-inline-functions -unroll=0 -traceback ${DEFINES} -c $<
 
 XMPI_OBJ = xmpi_univ.o
+xmpi_univ.o : timeprof.o
 
 EIGSYSTM_OBJ = eigsystm-lapack.o
+eigsystm-lapack.o : timeprof.o
 #EIGSYSTM_OBJ = eigsystm-scalapack.o
+#eigsystm-scalapack.o : timeprof.o
 
 FFT3D_OBJ = fft3d_mkl.o
 mkl_dfti.o: mkl_dfti.f90
 mkl_dfti.f90:
 	cp ${MKLROOT}/include/mkl_dfti.f90 .
 FFTGRID_H_SRC = fftgrid-scl.h
+fft3d_fftw.o : timeprof.o
 
 #FFT3D_OBJ = fft3d_fftw.o
 #FFTGRID_H_SRC = fftgrid-scl-fftw.h
+#fft3d_fftw.o : timeprof.o
 
 exchange_kern_cuda.o: exchange_kern_cuda.cu exchange_kern_cuda.h
 	nvcc -ccbin=icc -Xcompiler -openmp -I /usr/local/cuda40/cuda/NVIDIA_GPU_Computing_SDK/C/common/inc -m64 -c -arch sm_20 exchange_kern_cuda.cu
@@ -473,8 +478,10 @@
 arch_ha8k.o: arch_ha8k.f90 timeprof.o
 arch_exp_r128.o: arch_exp_r128.f90 timeprof.o
 
-subr_input_nompi.o: subr_input.F90 util.o
+subr_distpart.o : bsplpot.o
+subr_input_nompi.o: subr_input.F90 util.o read_tappinput.o
 	${FC} -DNO_MPI -o subr_input_nompi.o -c subr_input.F90
+subr_shrtor.o : meshgen.o
 
 ################## FOR ESM METHOD by M.Otani 2006/02/06####################
 ################## Modified by Y.Ando 2011/01/21 ##########################
@@ -534,6 +541,10 @@
 .F.f:
 	${CPP} -P ${DEFINES} $< | sed '/^ *$$/d' > $*.f
 
+install: default
+	install -d $(DESTDIR)/usr/bin
+	install inipot cgmrpt mdrpt vbpef vbstm wfn2chg wannier pefcos xticonv strconv diffstr iplstr cmpstr pe2dos tetrapdos hmatr2bnd atmcdist distpart $(DESTDIR)/usr/bin
+
 # Local Variables: ***
 # mode:Makefile ***
 # End:***
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -3,9 +3,11 @@
 	example-pt-111-co-atmeng.pdf \
 	formalism.pdf inputformat.pdf programs.pdf tutorial.pdf
 
-clean:
+clean distclean:
 	rm -f *.aux *.dvi *.log *.toc *.pdf
 
+install:
+
 develop.pdf:
 	platex --kanji=euc develop.tex
 	platex --kanji=euc develop.tex
